### PR TITLE
Add Tech-Angels profile

### PR DESCRIPTION
**Pull request by gravis**
refs #133

### CODE REVIEW

File: company-profiles/tech-angels.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+recently to [Go](http://golang.org) and [Angular.js](https://angularjs.org/).
+We are relying on [openshift](https://github.com/openshift/origin) (/docker) and vsphere for our infrastructure, and [PostgreSQL](http://postgresql.org). 
+
+## Office Locations
+
+Our main office is in Paris (France) and we have a subsidiary in Florida.
+
+## How to apply
+
+Send your application to am9ic0B0ZWNoLWFuZ2Vscy5jb20=
```
Comment (jessicard) [inline company-profiles/tech-angels.md]: Is this supposed to be a URL?

File: company-profiles/tech-angels.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Tech-Angels is a Services and Software company. 
+We excel in web and software development, and creating innovating services using Go (http://golang.org) + Angularjs.
+
+Tech-Angels features 3 kind of activities:
+
+* Consulting (engineering, code, audits, etc.)
+* Hosting 
+* SaaS
+
+We're the guys behing https://gemnasium.com
```
Comment (jessicard) [inline company-profiles/tech-angels.md]: "behing" typo :)

File: company-profiles/tech-angels.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Company size
+
+We want to keep the company small, to remain experts in what we do. 
+We are currently less than 10, and will grow to 20 in the future.
+
+## Remote status
+
+Tech-Angels doesn't have any office. We are all working remotly, except the ones behing on clients premises.
```
Comment (jessicard) [inline company-profiles/tech-angels.md]: "remotly" should be "remotely"

**Comment by jessicard** [comment]
Thanks! Added some comments. Will merge when fixed!

File: company-profiles/tech-angels.md (hunk 1 above)
Comment (gravis) [inline company-profiles/tech-angels.md]: Yes, that an encoded email address. Applicants will figure out how to decode it ;)

**Comment by gravis** [comment]
Thanks for your comments! I have addressed them, could you please take a look?

**Comment by jessicard** [comment]
Thank you!

**Pull request merged**